### PR TITLE
adding courtId only if case is in draft stage

### PR DESCRIPTION
**Pull request by TusharKatiyar2002**
https://github.com/pucardotorg/dristi/issues/2223 https://github.com/pucardotorg/dristi/issues/2282

## Summary by CodeRabbit

- **New Features**
	- Enhanced form configuration for complainant and witness age representation.
	- Improved document upload handling and validation processes for case management.
  
- **Bug Fixes**
	- Addressed validation and error handling for document uploads and duplicate entries.

- **Refactor**
	- Streamlined code for better readability and maintainability regarding case updates and validations.

### CODE REVIEW

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js [hunk 1]
```diff
@@ -382,9 +382,9 @@ export const reviewCaseFileFormConfig = [
                   value: "emails.emailId",
                 },
                 {
-                  type: "date",
-                  label: "DATE_OF_BIRTH",
-                  value: "witnessDateOfBirth",
+                  type: "text",
```
Comment (anirudh-0) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js:385]: intentional change ?

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ export const reviewCaseFileFormConfig = [
                 {
                   type: "text",
                   label: "AGE",
-                  value: "complainantVerification.complainantAge",
+                  value: "complainantAge",
```
Comment (anirudh-0) [inline frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js:34]: intentional change ?

**Review by anirudh-0 — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js (hunk 1 above)
Comment (TusharKatiyar2002) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js:385]: yes

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js (hunk 2 above)
Comment (TusharKatiyar2002) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/citizen/FileCase/Config/reviewcasefileconfig.js:34]: yes

**Pull request merged**